lib/rubocop/cop/autocorrect_logic.rb

```diff
@@ -19,7 +19,7 @@ def support_autocorrect?
       def autocorrect_enabled?
         # allow turning off autocorrect on a cop by cop basis
         return true unless cop_config
-        cop_config['AutoCorrect'] != 'False'
+        !['False', false].include?(cop_config['AutoCorrect'])
```

Comment:
I guessed as much, but I doubt many people are using this, so changing it won't be a big problem. We might do some checking for keys that shouldn't be strings in the future and warn the users when loading the config.